src/main/java/build/buildfarm/server/ContentAddressableStorageService.java

```diff
@@ -112,11 +112,14 @@ public void onSuccess(FindMissingBlobsResponse.Builder builder) {
               long elapsedMicros = stopwatch.elapsed(MICROSECONDS);
               logger.log(
                   requestLogLevel,
-                  format(
-                      "FindMissingBlobs(%s) for %d blobs in %gms",
-                      instance.getName(),
-                      request.getBlobDigestsList().size(),
-                      elapsedMicros / 1000.0));
+                  new StringBuilder()
+                    .append("FindMissingBlobs(")
+                    .append(instance.getName())
+                    .append(") for ")
+                    .append(request.getBlobDigestsList().size())
+                    .append(" blobs in ")
+                    .append(elapsedMicros / 1000.0)
```

Comment:
.append("ms")

Reply:
Yes, the nexts step will be to reduce most of INFO type logging to FINE and send metrics based logging elsewhere.

Reply:
I'm going to leave it as is since we have log parsing that will break if we modify the content. This will be going away in favor of actual metrics sending soon anyways.